packages/ui/src/components/input/input.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+
+interface InputProps {
+  className?: string;
+  id: string;
+  type?: string;
+  value?: string;
+  onChange?: (value: string) => void;
+  onBlur?: () => void;
+}
+
+export default forwardRef<HTMLInputElement, InputProps>(
+  function Input(props, ref) {
+    const { className, id, onBlur, onChange, type, value } = props;
+
+    const handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
+      e.preventDefault();
+      onChange && onChange(e.target.value);
+    };
+
+    const handleOnWheel = (e: React.WheelEvent<HTMLInputElement>) => {
+      //prevents imput from scrolling
```

Comment:
nit: Typo in "input".

Also, what does this comment mean by "preventing from scrolling"?

Reply:
Seems like the default behavior when input is focused. User can use mousewheel to change the value
Snippet above disables that feature
https://github.com/hackforla/lucky-parking/assets/66423127/a2d4b9b7-ab52-464c-ac58-ce3014500ff0

Reply:
Oh, got it. Great catch.